src/config/italiaConfig.js

```diff
@@ -388,5 +388,33 @@ export default function applyConfig(voltoConfig) {
   // comment out the following line and add the leadimage behavior in Document.xml file
   delete config.blocks.blocksConfig['leadimage'];
 
+  // TOC block anchors not working, customizing tocEntry
+  // to also return draftJS block id
+
```

Comment:
```suggestion
  config.settings.slate = {
    ...config.settings.slate ?? {},
    topLevelTargetElements: ['h1', 'h2', 'h3', 'h4', 'h5', 'h6'],
  };
```
So che slate qui sul tema base funziona, ma dato che nei temi specifici non funziona (per ora), metterei questa regola base così almeno questa in tutti c'è e non dobbiamo fare una PR in ogni tema specifico. Ci vedi problemi in questo cambio?

Reply:
non penso ci siano grossi problemi, ma ovviamente devo testare, e bisogna vedere se effettivamente funziona questa cosa. Committo la suggestion, poi passo in seconda puntata a provare sui temi figli tipo Parma